### DISCUSSION TITLE
What does "[h:a]" in the status bar mean?

### DISCUSSION THREAD

**Discussion by jingchengx** [ahrm/sioyek#1095] [General]
I went through the documentation and didn't find what it was for...

**Comment by ahrm** [comment]
It means the current highlight type is 'a'. For example used with `add_highlight_with_current_type` command among other things.

**Comment by Ascyii** [comment]
I would wish to see a feature to disable this info in the statusbar :)

**Reply by ahrm** [reply]
In the development branch you can customize the statusbar completely using the `status_bar_format` config variable.
